araproc/analysis/correlation.py

```diff
@@ -0,0 +1,114 @@
+import ROOT
+import numpy as np
+import math
+import os
+
+# HELP HERE: constants that we might want to store elsewhere, or can I retrieve them somehow?
+channel_num = 16
+
+def get_surface_correlation_ratio(corr_map, station_id, z_thresh):
```

Comment:
Can you give `z_thresh` a default value of -10 m? Also we may need to be careful the units of the radius in `corr_map` are the same as `z_thresh`.